esmf_regrid/__init__.py

```diff
@@ -1,4 +1,4 @@
 from .schemes import *
 
 
-__version__ = "0.5.dev0"
+__version__ = "0.5"
```

Comment:
Perhaps this should be 
```suggestion
__version__ = "0.5.0"
```
to be consistent with the last release?